src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindSmallIndelRegions.java

```diff
@@ -117,6 +122,9 @@ public void test( final GATKRead read, final List<BreakpointEvidence> evidenceLi
                 } else {
                     curContig = read.getContig();
                     curEnd = read.getStart() + BLOCK_SIZE;
+                    // clear 1st-half window when we switch contigs
+                    final int oldIdx = fillIdx ^ 1;
+                    libraryToHistoPairMap.forEach((lib,histPair) -> histPair[oldIdx].clear());
```

Comment:
IMO it would be clearer to select only the values in the map so it's more apparent that the keys aren't used, e.g.
```
libraryToHistoPairMap.values().forEach(histPair -> histPair[oldIdx].clear());
```